geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/AccumuloDescribeCommand.scala

```diff
@@ -0,0 +1,33 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.accumulo.commands
+
+import com.beust.jcommander.{JCommander, Parameters}
+import com.typesafe.scalalogging.LazyLogging
+import org.locationtech.geomesa.tools.accumulo.GeoMesaConnectionParams
+import org.locationtech.geomesa.tools.accumulo.commands.AccumuloDescribeCommand._
+import org.locationtech.geomesa.tools.common.FeatureTypeNameParam
+import org.locationtech.geomesa.tools.common.commands.{Command, DescribeCommand}
+
+
+class AccumuloDescribeCommand(parent: JCommander)
+  extends Command(parent)
+    with DescribeCommand
+    with CommandWithAccumuloDataStore
```

Comment:
what if you just had CommandWithAccumuloDatastore extend Command so you don't have to mix it in?